src/gwt/src/org/rstudio/core/client/SafeHtmlUtil.java

```diff
@@ -21,6 +21,67 @@
 
 public class SafeHtmlUtil
 {
+   public static class TagBuilder
```

Comment:
Note to self: GWT has its own HTML builders (e.g. see the `DivBuilder` class) and we can probably just use that instead.